soccer_predict/pipeline.py

```diff
@@ -117,17 +120,69 @@ def pipeline(scaler='StandardScaler',
         sc = getattr(__import__('sklearn.preprocessing', fromlist=[scaler]), scaler)
     except:
         return f"scaler:{scaler} inconnu"
-    
+
     try:
         md = getattr(__import__(model_path, fromlist=[model_name]), model_name)
     except:
         return f"model {model_name} ou {model_path} inconnu"
-    
+
     if classification:
         return make_pipeline(sc(), md())
     else:
         return make_pipeline(sc(), CustomMultiouputwrapper(md()),Posttreat())
-    
+
+
+def pipeline_scalers(scaler='StandardScaler',
+                     model_path='sklearn.linear_model',
+                     model_name='LinearRegression',
+                     classification=False):
```

Comment:
Il faudrait modifier les paramètres de la fonction (et le docstring en dessous) car il n'est plus nécessaire d'importer un scaler.
```suggestion
def pipeline_scalers(
                     model_path='sklearn.linear_model',
                     model_name='LinearRegression',
                     classification=False):
```